app/authentication_views.py

```diff
@@ -76,3 +83,11 @@ def login():
     logger.info(
         "Handling '/login' route, user is not authenticated, rendering login.html")
     return render_template("forms/login.html", form=form)
+
+@login_required
+@app.route("/logout")
+def logout():
+    name = current_user.username
```

Comment:
why do you have to assign current user to name?

Reply:
I don't have to. I have done it only for logging purpose.

Reply:
ok, can you rename name to something like loggedOutUsername?
this one confused me a bit, I had to check if it is new variable or some class member.
its because im python noob, but nicer name always helps